src/Generator/Passes/CheckIgnoredDecls.cs

```diff
@@ -367,6 +367,16 @@ private bool HasInvalidDecl(Declaration decl, out string msg)
             {
                 msg = "null";
                 return true;
+            }            
+
+            if (decl is Class)
```

Comment:
In general, do not use "is" + "as" because these are two type casts which is a little slower. Instead, use "as" + a null check. Please fix here and anywhere else you've used it this way.